tests/monotouch-test/CoreBluetooth/CentralManagerTest.cs

```diff
@@ -78,7 +78,7 @@ public void SetUp ()
 		{
 			if (TestRuntime.IsInCI && TestRuntime.CheckXcodeVersion (14, 0))
 				TestRuntime.AssertNotDesktop (); // Looks like this particular test doesn't like Desktop + M1 bot machines
-			// iOS 13 and friends require bluetooth permission
+												 // iOS 13 and friends require bluetooth permission
```

Comment:
This looks unrelated, maybe the autoformat bot is trying to be smarter heh

Reply:
@dalexsoto https://github.com/xamarin/xamarin-macios/pull/17524#discussion_r1104827753